### ISSUE TITLE
Multiple versioning URL not working

### ISSUE THREAD

**Issue by shaair** [domaindrivendev/Swashbuckle.AspNetCore#1513]
Multiple versioning was working with Swashbuckle.AspNetCore 4.0.1 but not working in 5.0.0. like

../api/api-docs ---not working
../api/api-limitted-docs ---working

```
app.UseSwaggerUI(options =>
            {
                options.RoutePrefix = "api-docs";
                string swaggerJsonBasePath = string.IsNullOrWhiteSpace(options.RoutePrefix) ? "." : "..";
                options.SwaggerEndpoint($"{swaggerJsonBasePath}/swagger/all/swagger.json", "v1");
            });
app.UseSwaggerUI(options =>
            {
                options.RoutePrefix = "api-limitted-docs";
                string swaggerJsonBasePath = string.IsNullOrWhiteSpace(options.RoutePrefix) ? "." : "..";
                options.SwaggerEndpoint($"{swaggerJsonBasePath}/swagger/limitted/swagger.json", "v1");
            });
```

**Comment by adamjones1** [comment]
This is a big problem for me too.

**Comment by domaindrivendev** [comment]
I believe this has been fixed since 5.0.0. It should work with the latest version (`5.5.0` at time of writing). Please pull that down and let me know how it goes

**Comment by adamjones1** [comment]
Confirmed this is fixed for me in the latest package - thanks!

**Comment by shaair** [comment]
> I believe this has been fixed since 5.0.0. It should work with the latest version (`5.5.0` at time of writing). Please pull that down and let me know how it goes

Yes i was not fixed in 5.0. Lets try with latest version.

**Comment by shaair** [comment]
Yes i was not fixed in 5.0. Lets try with latest version if its working.
